### PR TITLE
Adds OpenSearch provider

### LINKED ISSUE #2619: Add Habitats package search engine to the browser
Hi there :)

screenshot_20170617_081937

Firefox provides us with the feature to add search engines to its search bar.
The habitat package search engine does not support that now. 

I guess the implementation counts for other browsers then too.

### PR DESCRIPTION
**Pull request by rsertelon**
This provider can be added from www.habitat.sh / www.habitat.sh/blog / bldr.habitat.sh

Having the ability to add the provider "from anywhere" forced me to duplicate the `opensearch.xml` file.

Closes #2619

Signed-off-by: Romain Sertelon <romain@sertelon.fr>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-web/opensearch.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
+                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+  <ShortName>habitat.sh</ShortName>
```
Comment (ryankeairns) [inline components/builder-web/opensearch.xml]: Overall this looks great, thanks for adding it. Would you mind changing the ShortName and Description to:
ShortName: Habitat Search
Description: Use bldr.habitat.sh to search for packages on Habitat Builder.

Thanks!

Comment (rsertelon) [reply components/builder-web/opensearch.xml]: Will do, np!

Comment (rsertelon) [reply components/builder-web/opensearch.xml]: @ryankeairns done!

Comment (ryankeairns) [reply components/builder-web/opensearch.xml]: Thank you!

**Pull request merged**